lua/conform/formatters/pymarkdownlnt.lua

```diff
@@ -0,0 +1,11 @@
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/jackdewinter/pymarkdown",
+    description = "PyMarkdown is primarily a Markdown linter",
+  },
+  command = "pymarkdownlnt",
+  args = { "fix", "$FILENAME" },
+  exit_codes = { 0, 3 }, -- code 0 is given when trying a file that includes non-autofixable errors, code 3 when fixing is successfully applied
```

Comment:
What do you think of using [--return-code-scheme](https://pymarkdown.readthedocs.io/en/latest/user-guide/#-return-code-scheme-observability) instead of supplying special success exit codes?

Reply:
This is how the *default* scheme already behaves (returns 3 when it has fixed at least a file). Would you prefer to switch to the *minimal* scheme?

Reply:
Here you go :smile: